scripts/zones/Chamber_of_Oracles/mobs/Retiarius_XI-XIX.lua

```diff
@@ -1,14 +1,64 @@
 -----------------------------------
--- Area: Chamber of Oracles
---  Mob: Retiarius XI-XIX
--- BCNM: Legion XI Comitatensis
+-- Area : Chamber of Oracles
+-- Mob  : Retiarius XI-XIX
+-- BCNM : Legion XI Comitatensis
+-- Job  : Black Mage
 -----------------------------------
 mixins = { require('scripts/mixins/job_special') }
 -----------------------------------
 ---@type TMobEntity
 local entity = {}
 
-entity.onMobDeath = function(mob, player, optParams)
+local enfeebleTable =
+{
+    [1] = { xi.magic.spell.FROST,       xi.effect.FROST     },
+    [2] = { xi.magic.spell.DROWN,       xi.effect.DROWN     },
+    [3] = { xi.magic.spell.RASP,        xi.effect.RASP      },
+    [4] = { xi.magic.spell.POISONGA_II, xi.effect.POISON    },
+    [5] = { xi.magic.spell.BIO_II,      xi.effect.BIO       },
+    [6] = { xi.magic.spell.BLIND,       xi.effect.BLINDNESS },
+    [7] = { xi.magic.spell.SLEEPGA,     xi.effect.SLEEP_I   },
+    [8] = { xi.magic.spell.SLEEPGA_II,  xi.effect.SLEEP_II  },
```

Comment:
`[8] = { xi.magic.spell.SLEEPGA_II, xi.effect.SLEEP_I },`
xi.effect.SLEEP_II is unused